pkg/server/server.go

```diff
@@ -400,7 +400,7 @@ func CreateKHIServer(inspectionServer *inspection.InspectionTaskServer, serverCo
 			}
 			serverConfig.UploadFileStore.SetResultOnCompletedUpload(token, nil)
 
-			ctx.String(http.StatusOK, "ok")
+			ctx.String(http.StatusOK, "")
```

Comment:
Why did you change here?

Reply:
I don't wanted to create a new endpoint with responding something not in JSON.
We would need to define a schema for returning an error or just succeeded respond later.